aptihramy-backend/main.py

```diff
@@ -0,0 +1,38 @@
+from fastapi import FastAPI
+import uvicorn
+from database import Database
+import polars as pl
+import blitzbeaver as bb
+import time as time
+from database_config import (
+    RECORD_SCHEMA,
+    TRACKING_CONFIG,
+    PATH_GRAPH,
+    CSV_PATH,
+    START_YEAR,
+    END_YEAR,
+)
+
+
+def create_beaver_file(
+    record_schema: bb.RecordSchema,
```

Comment:
I would not include this function, as we don't expect users to run once the tracking process with the default (& hardcoded) config.